blacksmith/src/lib.rs

```diff
@@ -131,12 +131,14 @@ impl Blacksmith {
             ("rustc-diagnostic-code.html", "/compiler/diagnostic-codes.html"),
             ("rustc-team-maintenance.html", "/infra/team-maintenance.html"),
             ("stabilization-guide.html", "https://rust-lang.github.io/rustc-guide/stabilization_guide.html"),
-            ("stabilization-guide.html", "https://rust-lang.github.io/rustc-guide/stabilization_guide.html"),
             ("state-of-rust.html", "https://github.com/rust-lang/rust/projects/8"),
             ("test-suite.html", "https://rust-lang.github.io/rustc-guide/tests/intro.html"),
             ("toolstate.html", "/infra/toolstate.html"),
             ("triage-procedure.html", "/release/triage-procedure.html"),
             ("x-py.html", "https://rust-lang.github.io/rustc-guide/building/how-to-build-and-run.html"),
+            ("bug-fix-procedure.html", "https://rust-lang.github.io/rustc-guide/bug-fix-procedure.html"),
```

Comment:
This actually won't be enough since the current file is located at `compiler/bug-fix-procedure.html`. Could add something like the following code to the loop that creates the files? This would ensure if someone adds a path contained in a folder we can still properly redirect to new pages.

```rust
if let Some(parent) = Path::from(filename).parent() {
    std::fs::create_dir_all(parent).unwrap();
}
```